src/platform_impl/apple/appkit/window_delegate.rs

```diff
@@ -592,6 +594,14 @@ fn new_window(
         if attrs.platform_specific.title_hidden {
             window.setTitleVisibility(NSWindowTitleVisibility::NSWindowTitleHidden);
         }
+        if attrs.platform_specific.borderless_game {
+            let app = NSApplication::sharedApplication(mtm);
+            let presentation_options =
+                NSApplicationPresentationOptions::NSApplicationPresentationHideDock
+                    | NSApplicationPresentationOptions::NSApplicationPresentationHideMenuBar;
+            app.setPresentationOptions(presentation_options);
```

Comment:
I know I said something else before, but I think it might need to be set inside `window:willUseFullScreenPresentationOptions:` to work properly?

Reply:
Which also means that it's a dynamic property, i.e. something that the user could change at runtime, so perhaps it should be a property on `WindowExtMacOS`?

Reply:
I implemented it in both `WindowExtMacOS` and `WindowAttributesExtMacOS`.